package.json

```diff
@@ -30,7 +30,7 @@
   "scripts": {
     "dev": "nodemon",
     "start": "ts-node src/index.ts",
-    "test": "jest"
+    "test": "jest --runInBand"
```

Comment:
ensures using `npm test` will run each test suite sequentially so we don't have it trying to create two database connections at the same time or whatever multithreading thing you call it